src/tag/tag.stories.ts

```diff
@@ -42,6 +42,12 @@ storiesOf("Components|Tag", module)
 				title="Filter"
 				closeButtonLabel="Clear"
 			>filter</ibm-tag-filter>
+			<ibm-tag-filter
+				type="blue"
+				title="Disabled Filter"
```

Comment:
```suggestion
				title="Disabled filter"
```

Carbon Design System uses sentence capitalization for strings.